.taskcluster.yml

```diff
@@ -101,199 +107,212 @@ tasks:
                           $if: 'tasks_for in ["github-push", "github-release", "action", "cron"] && repoUrl == "https://github.com/mozilla-mobile/fenix"'
                           then: '3'
                           else: '1'
+
+                      short_head_branch:
+                          $if: 'head_branch[:11] == "refs/heads/"'
+                          then: {$eval: 'head_branch[11:]'}
+                          else: ${head_branch}
```

Comment:
What do you expect this to look like if it does not match `refs/heads/`? I think it may make sense to have this not have a value in that case, rather than have some value that may or may not have `/` in it.

Reply:
You're right, `refs/tags/` and `refs/pull` also exist and I don't think we want to extract a branch name out of them. I'm removing the `else` statement, then!